modules/hrm/includes/class-ajax.php

```diff
@@ -1269,7 +1278,7 @@ public function birthday_wish() {
     public function employee_update_performance() {
         // check permission for adding performance
         if ( ! isset( $_REQUEST['_wpnonce'] ) || ! wp_verify_nonce( sanitize_text_field( wp_unslash( $_REQUEST['_wpnonce'] ) ), 'wp-erp-hr-nonce' ) ) {
-            //$this->send_error( __( 'Error: Nonce verification failed', 'erp' ) );
+            $this->send_error( __( 'Error: Nonce verification failed', 'erp' ) );
```

Comment:
this function is called from three different ajax hooks, have you tested all of them? maybe there was a reason behind disabling nonce check, kindly make sure all related ajax request are wroking correctly.